src/sys/resource.rs

```diff
@@ -0,0 +1,49 @@
+use std::mem;
+
+use libc::{self, c_int};
+pub use libc::{rlimit, RLIM_INFINITY};
```

Comment:
What about `RLIM_SAVED_CUR` and `RLIM_SAVED_MAX`?

Why don't we wrap rlimit and provide proper accessor methods? I admit that it would be purely cosmetic in this case, as far as I can see.